content/en/docs/architecture/bugzilla.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Bugzilla Integration"
+description: An overview of the integration of Bugzilla with the Openshift CI
+---
+
+## Linking Pull Requests to Bugzilla bugs
+
+Pull requests can reference a Bugzilla bug in their title, for example
+`Bug 123: Rest of the PR title`. This makes the bugzilla plugin run a couple
+of checks:
+* If a bug with the id `123` exists and is in state `open`
+* If the target release of the bugzilla bug matches the configured target release for the branch the PR points at. This mapping is
+	configured in the [openshift/release repository][0] via the `target_release` property
+* If the bug is in the correct state. Accepted states are configured in the [openshift/release repository][0] through the `valid_states`
```

Comment:
i'm really happy to see you linked to an example, but i think people may miss this link because it's specifically tied to talking about the valid_states property.  I would include a direct mention of "for an example of how the plugin is configured for a specific repository, see:" and link to an example of that (not the default config) that illustrates the various config properties.